package.json

```diff
@@ -183,23 +182,19 @@
     "danger-plugin-no-test-shortcuts": "^2.0.0",
     "enzyme": "^3.11.0",
     "enzyme-adapter-react-16": "^1.15.2",
-    "eslint": "^5.16.0",
+    "eslint": "^6.8.0",
     "eslint-config-prettier": "^6.9.0",
-    "eslint-config-standard": "^12.0.0",
```

Comment:
It's not used anywhere but maybe that's a mistake?

Reply:
We use `standard-react` –  does it declare its own dependency on `standard`?

Reply:
Ah yes: https://github.com/badges/shields/pull/4489#discussion_r363082988

Reply:
@paulmelnikow When I checked, seemed `eslint-config-standard-react` was NOT bringing `eslint-config-standard`. I'm not too familiar with it so not sure what to do at this point.

Reply:
@paulmelnikow I just checked and `eslint-config-standard` is not enforced right now in the codebase, if we activate it, we get many errors all over the place. Most of it can be automatically fixed with `--fix` but then the syntax is not compatible with prettier rules (`space-before-function-paren`, `comma-dangle`).

Reply:
eslint-config-prettier should disable those, maybe check the order in which those two are listed in extends?

Reply:
Ah, you must have solved that.

Reply:
I adjusted the config in https://github.com/badges/shields/pull/4489/commits/4bd3ec8abf4bc58c479b1aa3f08dbe9ce64f3808